src/backend/distributed/commands/extension.c

```diff
@@ -769,13 +769,23 @@ RecreateExtensionStmt(Oid extensionOid)
 
 	/* make DefEleme for extensionSchemaName */
 	Node *schemaNameArg = (Node *) makeString(extensionSchemaName);
-
 	DefElem *schemaDefElement = makeDefElem("schema", schemaNameArg, location);
 
 	/* append the schema name DefElem finally */
 	createExtensionStmt->options = lappend(createExtensionStmt->options,
 										   schemaDefElement);
 
+	char *extensionVersion = get_extension_version(extensionOid);
+	if (extensionVersion != NULL)
```

Comment:
This was a bug exposed by the create_object_propagation change. We did not include the version in the CREATE EXTENSION, which caused inconsistency between coordinator and workers.